src/pages/index.js

```diff
@@ -82,31 +142,82 @@ const editFormValidator = new FormValidator(
 );
 const addFormValidator = new FormValidator(validationSettings, addCardModal);
 
+const editAvatarValidator = new FormValidator(
+  validationSettings,
+  avatarEditModal
+);
+
 //Initialize all instances
-cardSection.renderItems(initialCards);
+api.getInitialCards().then((cards) => {
+  cardSection.renderItems(cards);
+});
+
 cardPreviewPopup.setEventListeners();
 editPopup.setEventListeners();
 addPopup.setEventListeners();
+editAvatarPopup.setEventListeners();
 editFormValidator.enableValidation();
 addFormValidator.enableValidation();
+editAvatarValidator.enableValidation();
 
 /*--------------------------------------------------*/
 /*                      Functions                   */
 /*--------------------------------------------------*/
 
+function handleDeleteCardClick() {
+  confirmDeleteModal.open();
+  confirmDeleteModal.setSubmitAction(() => {
+    const id = this.getId();
+
+    api
+      .removeCard(id)
+      .then((res) => {
+        this.handleDeleteCard();
```

Comment:
`this.`


`this` should **not** be used outside classes in functions in `index.js` . This context is easy to lose. You need to find other ways of passing data.